src/style.css

```diff
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./Avenir.ttc);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+}
+
+.box {
+  position: relative;
+  margin: 20px 0;
+}
+
+.inp {
+  font-family: inherit;
+  width: 100%;
+
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.inp:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.inp:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+}
+
+.bin {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+
+}
+
+.sin {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.imgbig {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.imgsmall {
+  max-width: 25%;
+  height: 25%;
```

Comment:
strange percents why do you need them here?

Reply:
I took the original picture search.svg and scale it for needed size :) but height in this case is enough

Reply:
better to use pixels here

Reply:
ok